CMakeLists.txt

```diff
@@ -2,8 +2,13 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.3)
 project(polygames)
 
 set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_FLAGS
-    "${CMAKE_CXX_FLAGS} -fsized-deallocation -O3 -ffast-math")
+#set(CMAKE_CXX_FLAGS
```

Comment:
We will need to remember to set these back once things start working

Reply:
Sure I'll add conditional ones.

We don't need -fsized-deallocation https://developer.arm.com/documentation/101754/0619/armclang-Reference/armclang-Command-line-Options/-fsized-deallocation---fno-sized-deallocation